pkgs/development/haskell-modules/lib/default.nix

```diff
@@ -354,4 +354,32 @@ rec {
   # same package in the (recursive) dependencies of the package being
   # built. Will delay failures, if any, to compile time.
   allowInconsistentDependencies = compose.allowInconsistentDependencies;
+
+
+  # Collect all the cabal2nix derivers (i.e. nix file generated from a call
```

Comment:
```suggestion
  # Collect all the cabal2nix derivers (i.e. nix files generated from a call
```

Reply:
I removed that part of the changes, so this is no longer relevant: I'm checking "Resolved".